torchaudio/datasets/cmudict.py

```diff
@@ -0,0 +1,92 @@
+from collections import OrderedDict
+import os
+import re
+from pathlib import Path
+from typing import Tuple, Union, List, Dict
+
+from torch.utils.data import Dataset
+from torchaudio.datasets.utils import download_url
+
+_CHECKSUMS = {
+    "http://svn.code.sf.net/p/cmusphinx/code/trunk/cmudict/cmudict-0.7b":
+    "825f4ebd9183f2417df9f067a9cabe86",
+    "http://svn.code.sf.net/p/cmusphinx/code/trunk/cmudict/cmudict-0.7b.symbols":
+    "385e490aabc71b48e772118e3d02923e",
+}
+
+
+class CMUDICT(Dataset):
```

Comment:
Can we call this `CMUDict`? I do not see why this has to be all capital. This is not a convention or a good practice to follow either.

Reply:
I actually like CMUDict better.
But I saw that many of the previous dataset names like [CommonVoice is called COMMONVOICE](https://github.com/pytorch/audio/blob/master/torchaudio/datasets/commonvoice.py#L32), [LJSpeech is called LJSPEECH](https://github.com/pytorch/audio/blob/master/torchaudio/datasets/ljspeech.py#L20) and so on.

Reply:
Yeah, let's call it `CMUDict`. That's more natural.